pkg/kubectl/cmd/create.go

```diff
@@ -45,7 +45,7 @@ $ cat pod.json | kubectl create -f -`
 func NewCmdCreate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	var filenames util.StringList
 	cmd := &cobra.Command{
-		Use:     "create -f FILENAME",
+		Use:     "create -f /path/to/FILENAME",
```

Comment:
I guess this is different from, say, delete because the FILENAME isn't optional, so it isn't in brackets?

Would it be hard to special-case "FILENAME"?